hirefire/procs/celery.py

```diff
@@ -5,6 +5,15 @@
 
 from celery.app import app_or_default
 
+try:
+    from librabbitmq import ChannelError
+except ImportError:
+    try:
+        from amqp.exceptions import ChannelError
+    except ImportError:
+        # No RabbitMQ API wrapper installed, different celery broker usedL:wq
```

Comment:
Oops. Looks like you're using vim :smirk:

Reply:
haha, good catch ;)